src/sentry/middleware/health.py

```diff
@@ -19,9 +21,16 @@ def process_request(self, request):
 
         from sentry.status_checks import check_all
         from sentry.utils import json
-        problems, checks = check_all()
 
+        results = check_all()
```

Comment:
we need to ensure that this doesnt return non-errors by default (its not clear to me if thats true skimming over this)